### ISSUE TITLE
Dragonflight Beta / PTR 10.0 errors

### ISSUE THREAD

**Issue by Voxxel** [frozn/TipTac#101]
Hi there, I'm very happy to found out you make updates for my favorite addon. I hope you'll update TipTac for Dragonflight too. The PTR is up for 10.0 if you're missing the beta access. I tried out your latest version and it's mostly working, however I received some LUA errors as well as the returning bug of the white, semi-transparent tooltip background that turns into the set black over some activity.
Would you take a look for fixing TipTac to DF too please?

```
1x Interface/AddOns/TipTac/ttCore.lua:1846: attempt to index field 'ListScrollFrame' (a nil value)
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1846: in function `ApplyHooksToCFMLLSF'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1943: in function `?'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:538: in function <Interface/AddOns/TipTac/ttCore.lua:538>
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface/FrameXML/UIParent.lua"]:530: in function `UIParentLoadAddOn'
[string "@Interface/FrameXML/UIParent.lua"]:827: in function `Communities_LoadUI'
[string "@Interface/FrameXML/UIParent.lua"]:1128: in function `ToggleCommunitiesFrame'
[string "@Interface/FrameXML/UIParent.lua"]:983: in function `ToggleGuildFrame'
[string "@Interface/FrameXML/MainMenuBarMicroButtons.lua"]:791: in function <Interface/FrameXML/MainMenuBarMicroButtons.lua:789>
[string "=[C]"]: ?

Locals:
self = TipTac {
 BottomLeftCorner = Texture {
 }
 close = Button {
 }
 AddModifiedTip = <function> defined @Interface/AddOns/TipTac/ttCore.lua:2018
 MirrorAnchorsSmart = <table> {
 }
 CURSOR_UPDATE = <function> defined @Interface/AddOns/TipTac/ttCore.lua:404
 GetNearestPixelSize = <function> defined @Interface/AddOns/TipTac/ttCore.lua:589
 GetBackdropColor = <function> defined @Interface/SharedXML/Backdrop.lua:390
 RightEdge = Texture {
 }
 CVAR_UPDATE = <function> defined @Interface/AddOns/TipTac/ttCore.lua:516
 ClearBackdrop = <function> defined @Interface/SharedXML/Backdrop.lua:282
 SetBackdropColorLocked = <function> defined @Interface/AddOns/TipTac/ttCore.lua:757
 ADDON_LOADED = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1901
 SetBackdropBorderColorLocked = <function> defined @Interface/AddOns/TipTac/ttCore.lua:772
 ResetBackdropBorderColorLocked = <function> defined @Interface/AddOns/TipTac/ttCore.lua:787
 tipsToModify = <table> {
 }
 SetupTextureCoordinates = <function> defined @Interface/SharedXML/Backdrop.lua:214
 SendElementEvent = <function> defined @Interface/AddOns/TipTac/ttCore.lua:363
 HasBackdropInfo = <function> defined @Interface/SharedXML/Backdrop.lua:278
 GetEdgeSize = <function> defined @Interface/SharedXML/Backdrop.lua:188
 TopEdge = Texture {
 }
 ApplyUnitAppearance = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1162
 BottomRightCorner = Texture {
 }
 padding = <table> {
 }
 LeftEdge = Texture {
 }
 UI_SCALE_CHANGED = <function> defined @Interface/AddOns/TipTac/ttCore.lua:523
 SetBackdropLocked = <function> defined @Interface/AddOns/TipTac/ttCore.lua:750
 VARIABLES_LOADED = <function> defined @Interface/AddOns/TipTac/ttCore.lua:458
 SetBackdropBorderColor = <function> defined @Interface/SharedXML/Backdrop.lua:422
 ApplyHooksToTips = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1724
 text = FontString {
 }
 SetBackdrop = <function> defined @Interface/SharedXML/Backdrop.lua:329
 GetBackdropBorderColor = <function> defined @Interface/SharedXML/Backdrop.lua:409
 GetBackdrop = <function> defined @Interface/SharedXML/Backdrop.lua:347
 HookTips = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1884
 playerLevel = 70
 isColorBlind = false
 SetBorderBlendMode = <function> defined @Interface/SharedXML/Backdrop.lua:266
 CreatePushArray = <function> defined @Interface/AddOns/TipTac/ttCore.lua:337
 MirrorAnchors = <table> {
 }
 RegisterElement = <function> defined @Interface/AddOns/TipTac/ttCore.lua:357
 ApplyHooksToLFGLFAVSFB = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1868
 PLAYER_LEVEL_UP = <function> defined @Interface/AddOns/TipTac/ttCore.lua:422
 RemoveUnwantedLines = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1071
 elements = <table> {
 }
 ClassColorMarkup = <table> {
 }
 ApplyHooksToCFMLLSF = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1845
 Center = Texture {
 }
 OnBackdropSizeChanged = <function> defined @Interface/SharedXML/Backdrop.lua:182
 OnBackdropLoaded = <function> defined @Interface/SharedXML/Backdrop.lua:152
 AddLockingFeature = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1621
 GetUnitReactionIndex = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1101
 0 = <userdata>
 ApplySettings = <function> defined @Interface/AddOns/TipTac/ttCore.lua:659
 TopLeftCorner = Texture {
 }
 SetPaddingVariables = <function> defined @Interface/AddOns/TipTac/ttCore.lua:642
 ReApplyAnchorTypeForMouse = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1049
 ApplyBackdrop = <function> defined @Interface/SharedXML/Backdrop.lua:294
 AddTargetedBy = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1127
 ApplyTipBackdrop = <function> defined @Interface/AddOns/TipTac/ttCore.lua:873
 AnchorFrameToMouse = <function> defined @Interface/AddOns/TipTac/ttCore.lua:1041
 BottomEdge = Texture {
 }
 CURSOR_CHANGED = <function> defined @Interface/AddOns/TipTac/ttCore.lua:408
 SetPadding = <function> defined @Interface/AddOns/TipTac/ttCore.lua:796
 DISPLAY_SIZE_CHANGED = <function> defined @Interface/AddOns/TipTac/ttCore.lua:532
```
 
```
1x Interface/AddOns/TipTac/ttCore.lua:1145: attempt to index upvalue 'targetedByList' (a nil value)
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1145: in function `AddTargetedBy'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1203: in function `ApplyUnitAppearance'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1399: in function <Interface/AddOns/TipTac/ttCore.lua:1358>
[string "=[C]"]: ?
[string "=[C]"]: ?
``` 
 
```
4x Interface/AddOns/TipTac/ttCore.lua:1872: attempt to index field 'ScrollFrame' (a nil value)
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1872: in function `ApplyHooksToLFGLFAVSFB'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1791: in function `ApplyHooksToTips'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1887: in function `HookTips'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:500: in function `?'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:538: in function <Interface/AddOns/TipTac/ttCore.lua:538>
``` 
 
```
46x FontString:SetText(): Font not set
[string "=[C]"]: in function `SetText'
[string "@Interface/AddOns/TipTac/ttBars.lua"]:149: in function `SetFormattedBarValues'
[string "@Interface/AddOns/TipTac/ttBars.lua"]:256: in function `?'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:366: in function `SendElementEvent'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1172: in function `ApplyUnitAppearance'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1399: in function <Interface/AddOns/TipTac/ttCore.lua:1358>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetUnit'
[string "@Interface/FrameXML/UnitFrame.lua"]:469: in function `UnitFrame_UpdateTooltip'
[string "@Interface/FrameXML/UnitFrame.lua"]:459: in function <Interface/FrameXML/UnitFrame.lua:458>
``` 
 
```
115x Interface/AddOns/TipTac/ttCore.lua:1151: attempt to index upvalue 'targetedByList' (a nil value)
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1151: in function `AddTargetedBy'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1203: in function `ApplyUnitAppearance'
[string "@Interface/AddOns/TipTac/ttCore.lua"]:1399: in function <Interface/AddOns/TipTac/ttCore.lua:1358>
[string "=[C]"]: ?
[string "=[C]"]: ?
```

**Comment by frozn** [comment]
Thanks for reporting this! There're currently no realms online for PTR, but I'm on it...

**Comment by Voxxel** [comment]
Have you disabled the mouse-anchoring feature too? Because I can't get it to work, the tooltips are always in the stock position in the right bottom corner.

**Comment by frozn** [comment]
Hm, I can't reproduce this. Mouse anchoring for world/frame units and frame tips is working fine in df. :raised_eyebrow:

**Comment by Voxxel** [comment]
I'm using the DF Beta btw. Turned off other addons, only TipTac and TipTacOptions are enabled, in the options the anchor set to right upper of cursor.

The TipTac version I have is:
## Interface: 90207
## Version: 22.10.14

**Comment by frozn** [comment]
Holy %!$%$... on df ptr (wow version 10.0.0) everything works fine, but on df beta (wow version 10.0.2) there are many changes incoming regarding the GameTooltip, see here: https://wowpedia.fandom.com/wiki/Patch_10.0.2/API_changes#Tooltip_Changes

Uff... this will take time :exploding_head:

Currently I'm updating all my addons which I support to be compatible with ptr (10.0.0). I will look after it as soon as I finished.

**Comment by Voxxel** [comment]
wow, nice long feed!
I hope you'll have time for it. Thank you in advance!

**Comment by frozn** [comment]
Fixed all issues regarding wow patch 10.0.2 with release [v22.11.14](https://github.com/frozn/TipTac/releases/tag/v22.11.14-release).
